.github/actions/iam-drift-detection/action.yml

```diff
@@ -50,7 +50,9 @@ runs:
         ORGANIZATION_ID: '${{ inputs.organization_id }}'
         GCS_BUCKET_LABEL: '${{ inputs.gcs_bucket_label }}'
       run: |
-        diff_snippet="$(.guardian/iam-drift-detection/detect_drift.sh "$ORGANIZATION_ID" "$GCS_BUCKET_LABEL")"
+        driftignore_file="$(pwd)/.driftignore"
```

Comment:
Maybe make this DRIFTIGNORE_FILE just for the sake of the convention of capitalizing env vars?

Reply:
its not an env var though? Its just a bash variable. It would look weird to do that for `driftignore_file` and not for `diff_snippet`. wdyt?

Reply:
Sorry, I misspoke, what I mean is that we conventionally capitalize shell variables, of which environment variables are only a subset. So maybe diff_snippet and driftignore_file should both be capitalized.

I don't have strong feeling on it, if you have a strong preference for it to stay this way, that's fine with me.

Reply:
Consulted the googs conventions go/shell-style#naming-conventions

Seems as though constants, env variables, and variables that are exported should be upper cased. Everything else should be lower. They do add that constants determined at runtime should still be uppercased. I think in this situation the `driftignore_file` qualifies as a constant so I uppercased it.

Reply:
TIL of that style guide rule, thanks.